### PR TITLE
delete button

### PR DESCRIPTION
**Pull request by darkdarkdragon**
**What does this pull request do? Explain your changes. (required)**
- show 'Reveal your stream key and the full URL via the button above' text right of the RTMP ingest link
- disable 'Delete' button after delete

**Specific updates (required)**

- 
- 
- 

**How did you test each of these updates (required)**

**Does this pull request close any open issues?**

**Screenshots (optional):**

**Checklist:**

- [ ] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by darkdarkdragon** [comment]
image

File: packages/livepeer.com/www/pages/app/stream/[id].tsx [hunk 1]
```diff
@@ -199,11 +201,29 @@ export default () => {
...
+                    anchor={false}
+                  />
+                ) : (
+                  <Flex
+                    sx={{ justifyContent: "flex-start", alignItems: "center" }}
+                  >
+                    <Box
+                      sx={{ minWidth: 125, fontSize: 12, paddingRight: "1em" }}
+                    >
+                      {getIngestURL(stream, false)}
```
Comment (angyangie) [inline packages/livepeer.com/www/pages/app/stream/[id].tsx:216]: Think we might need to remove this line ? I'm seeing the stream key before I hit the `Show secret stream button`. Without this line I see correct expected behavior

Screen Shot 2020-06-22 at 12 00 06 PM

File: packages/livepeer.com/www/pages/app/stream/new-stream.tsx [hunk 2]
```diff
@@ -100,6 +101,7 @@ export default () => {
             <Box>Stream name</Box>
             <Box>
               <Input
+                autoFocus={true}
```
Comment (angyangie) [inline packages/livepeer.com/www/pages/app/stream/new-stream.tsx:104]: Cool! What does this do?

File: packages/livepeer.com/www/pages/app/stream/[id].tsx (hunk 1 above)
Comment (darkdarkdragon) [reply packages/livepeer.com/www/pages/app/stream/[id].tsx:216]: It should be here, but there was bug in `getIngestURL` function. Fixed.

File: packages/livepeer.com/www/pages/app/stream/new-stream.tsx (hunk 2 above)
Comment (darkdarkdragon) [reply packages/livepeer.com/www/pages/app/stream/new-stream.tsx:104]: Put input focus to the text field when 'create stream' page opens.

Comment (angyangie) [reply packages/livepeer.com/www/pages/app/stream/new-stream.tsx:104]: Niceee!

File: packages/livepeer.com/www/pages/app/stream/[id].tsx [hunk 3]
```diff
@@ -128,17 +130,16 @@ export default () => {
...
+    return ingest.length ? pathJoin(ingest[0].ingest, key) : key || "";
   };
   const getPlaybackURL = (stream: Stream): string => {
     return ingest.length
-      ? `${ingest[0].playback}/${stream.playbackId}/index.m3u8`
+      ? pathJoin(ingest[0].playback, `${stream.playbackId}/index.m3u8`)
       : stream.playbackId || "";
   };
   const tabs = getTabs(0);
+  const backLink = query.admin ? "/app/admin/streams" : "/app/user";
```
Comment (angyangie) [inline packages/livepeer.com/www/pages/app/stream/[id].tsx:142]: Really minor, and only important for admin dashboard. Didn't notice this before. If we're getting to `stream/[id]` from admin, we should probably pull `getTabs` from "../admin"; and not from "../user", so we continue seeing the same subtab menu!

**Review by angyangie — APPROVED** [review]
Minor comment, but other than that, LGTM !

**Pull request merged**